modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,7 +3,109 @@
 // * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
 
 :_content-type: PROCEDURE
+
+###################################################
 [id="ztp-adding-new-content-to-gitops-ztp_{context}"]
+= Adding custom content to the {ztp} pipeline
+
+Perform the following procedure to add new content to the {ztp} pipeline.
+
+.Procedure
+
+. Create a subdirectory,  `source-crs`,  in the directory containing the `kustomization.yaml` for the `PolicyGenTemplate` CR. 
```

Comment:
Generally, you use multiple PGT CRs to manage various aspects of a fleet of clusters.